File: java/src/array/NumberExit.java
package array;

import java.util.Arrays;
import java.util.List;

public class NumberExit {
    public static void main(String[] args) {
        int number = 3;
        List<Integer> list = Arrays.asList(3, 5, 6, 1, 2, 4);

        System.out.println(isNumberExit(number, list));
    }

    static boolean isNumberExit(int number, List<Integer> list) {
        for (Integer i : list) {
            if (number == i) {
                return true;
            }
        }
        return false;
    }
}

File: python/array/number_exit.py
# 배열에서 특정 요소 찾기
# [3,5,6,1,2,4] - 특정 숫자 존재 True, 없으면 False

def is_number_exit(number, array):
    for a in array:
        if a == number:
            return True
    return False


if __name__ == "__main__":
    number = 9
    array = [3, 5, 6, 1, 2, 4]

    print(is_number_exit(number, array))
